Sources/Vapor/Multipart/FormDataDecoder.swift

```diff
@@ -75,9 +75,12 @@ private final class FormDataDecoderContext {
         case 2:
             let name = codingPath[0].stringValue + "[]"
             guard let offset = codingPath[1].intValue else {
-                throw MultipartError.nesting
+              throw MultipartError.nesting
```

Comment:
Nit - style guide is 4 spaces
```suggestion
                throw MultipartError.nesting
```